dss/util/bundles.py

```diff
@@ -0,0 +1,88 @@
+import json
+import typing
+
+import dss
+from dss import Config, DSSException
+from dss.hcablobstore import BundleFileMetadata, BundleMetadata
+from dss.util import UrlBuilder
+from cloud_blobstore import BlobNotFoundError
+
+def get_bundle_from_bucket(
+        uuid: str,
+        replica: str,
+        version: str,
+        bucket: str,
+        directurls: bool=False):
+
+    uuid = uuid.lower()
+
+    handle, hca_handle, default_bucket = Config.get_cloud_specific_handles(replica)
+
+    # need the ability to use fixture bucket for testing
+    bucket = default_bucket if bucket is None else bucket
+
+    if version is None:
+        # list the files and find the one that is the most recent.
+        prefix = "bundles/{}.".format(uuid)
```

Comment:
somewhat out of the scope of this diff, given that this is a copy, but this should be: `f"bundles/{uuid}"`

Reply:
this is actually correct, we are looking for a file with the the most recent version starting from the "bundles/uuid.<version>"

Reply:
I meant the f-style strings.